testsuite/integration/ws/src/test/java/org/jboss/as/test/integration/ws/authentication/EJBEndpointSecuredWSDLAccessTestCase.java

```diff
@@ -81,65 +83,70 @@ public void createService() throws Exception {
 
 
     @Test
-    public void accessWSDLWithValidUsernameAndPassord() throws Exception {
+    public void accessWSDLWithValidUsernameAndPassword() throws Exception {
         URL wsdlURL = new URL(baseUrl, "/jaxws-authentication-ejb3-for-wsdl/EJB3ServiceForWSDL?wsdl");
         String encoding = base64Encode("user1:password1");
 
-        DefaultHttpClient httpclient = new DefaultHttpClient();
-        HttpGet httpget = new HttpGet(wsdlURL.toString());
-        httpget.setHeader("Authorization", "Basic " + encoding);
-        HttpResponse response = httpclient.execute(httpget);
+        HttpResponse response;
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            HttpGet httpget = new HttpGet(wsdlURL.toString());
+            httpget.setHeader("Authorization", "Basic " + encoding);
+            response = httpClient.execute(httpget);
```

Comment:
CloseableHttpClient.execute(...) returns a CloseableHttpResponse.
To prevent resource leaks, the request pattern should look like:
```java
try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
   HttpUriRequest request = ...;
   try (CloseableHttpResponse response = client.execute(request)) {
      // Verify response
   }
}
```
or
```java
try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
   HttpUriRequest request = ...;
   client.execute(request, response -> {
      // Verify response
   });
}
```